src/main/java/com/revinate/sendgrid/model/User.java

```diff
@@ -0,0 +1,23 @@
+package com.revinate.sendgrid.model;
+
+public class User extends SendGridModel {
+
+    private String userName;
```

Comment:
I think SendGrid calls this "username", with a lowercase "n", instead of "userName". Could you check the value returned by the API to make sure?

Reply:
Yes, my bad. It's 'username'.